.github/workflows/pull-request.yml

```
@@ -272,6 +280,126 @@ jobs:
 
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
+    Start-App-Without-Docker:
```

actionlint (1.7.4)

unexpected key "Start-App-Without-Docker" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)